apps/src/codebridge/PopUpButton/PopUpButton.tsx

```diff
@@ -33,6 +34,7 @@ export const PopUpButton = ({
   const [dropdownStyles, setDropdownStyles] = useState<React.CSSProperties>({});
   const dropdownRef = useRef<HTMLDivElement | null>(null);
   const [updatedStyles, setUpdatedStyles] = useState(false);
+  const {theme} = useTheme();
```

Comment:
Wondering if it'd be worth leaving a comment here and other two places why we need `useTheme`. Thoughts?

Reply:
added!